Add New Featured Artist: Thank You Scientist newspost (#3603)
ppy/osu-wiki: news/2020-05-27-new-featured-artist-thank-you-scientist.md

---
layout: post
title: "New Featured Artist: Thank You Scientist"
date: 2020-05-27 08:00:00 +0000
---

**Thank You Scientist** adds some method to the madness as our latest addition to the Featured Artist lineup!

![](https://assets.ppy.sh/artists/83/header.jpg)

Our quest to encompass just about every genre and musical style continues this week with our latest snag, the seven-man wonderband that is [**Thank You Scientist**](https://osu.ppy.sh/beatmaps/artists/83). 

> Turning heads since 2012, the New Jersey based septet is widely renowned for their ecletic sound drawing upon influences from dozens of genres and folding them all under the progressive banner.

Reviewer: ```suggestion
Turning heads since 2012, the New Jersey-based septet is widely renowned for their eclectic sound that draws upon influences from dozens of genres and folds them all under the progressive banner.
```

> Though a little bit longer than you might be used to, their work fills a neat niche which has historically been well received, with mappers like [-Visceral-](https://osu.ppy.sh/users/1855540) (formerly known as Theaceae) and [Mun](https://osu.ppy.sh/users/6699165) turning heads with their map to [*My Famed Disappearing Act*](https://osu.ppy.sh/beatmapsets/765338) back in early 2018.

Reviewer: ```suggestion
Though a little bit longer than you might be used to, their work fills a neat niche which has historically been well-received, with mappers like [-Visceral-](https://osu.ppy.sh/users/1855540) (formerly known as Theaceae) and [Mun](https://osu.ppy.sh/users/6699165) turning heads with their map of [*My Famed Disappearing Act*](https://osu.ppy.sh/beatmapsets/765338) back in early 2018.
```

Don't take our word for it, give them a listen:

### My Famed Disappearing Act

Check out [this classic collaboration set](https://osu.ppy.sh/beatmapsets/765338) hosted by [-Visceral-](https://osu.ppy.sh/users/1855540)!

<audio controls>
    <source src="https://assets.ppy.sh/artists/83/previews/2171.mp3" type="audio/mpeg">
</audio>

### Mr. Invisible

Check out [this brand new set](https://osu.ppy.sh/beatmapsets/1130581) by [DTM9 Nowa](https://osu.ppy.sh/users/5428909) of Aspire fame!

<audio controls>
    <source src="https://assets.ppy.sh/artists/83/previews/2176.mp3" type="audio/mpeg">
</audio>

### Rube Goldberg Variations

<audio controls>
    <source src="https://assets.ppy.sh/artists/83/previews/2179.mp3" type="audio/mpeg">
</audio>

### Terraformer

<audio controls>
    <source src="https://assets.ppy.sh/artists/83/previews/2191.mp3" type="audio/mpeg">
</audio>

### FXMLDR

<audio controls>
    <source src="https://assets.ppy.sh/artists/83/previews/2186.mp3" type="audio/mpeg">
</audio>

---

> All of the above plus a whole bunch more are available right now over at [**Thank You Scientist**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/83), so make sure to check them out! Everything's all pre-timed and usual and just waiting for your creative touch, give mapping a try!

Reviewer: ```suggestion
All of the above plus a whole bunch more are available right now over at [**Thank You Scientist**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/83), so make sure to check them out! Everything's all pre-timed as usual and just waiting for your creative touch. Give mapping a try!
```
